addons/html_editor/static/src/core/clipboard_plugin.js

```diff
@@ -456,8 +456,6 @@ export class ClipboardPlugin extends Plugin {
             if (
                 (isParagraphRelatedElement(block) ||
                     this.dependencies.baseContainer.isCandidateForBaseContainer(block)) &&
-                // TODO specific exception for "PRE" to keep everything inside one PRE.
-                // Consider removing this if PRE is to be used as a paragraph.
                 block.nodeName !== "PRE" &&
```

Comment:
I'm surprised that this condition is still there considering the other ones that were related (notably in dom_plugin.js and dom_info.js) have been removed/put back.